### PR TITLE
[BEAM-2101] Allow anonymous access of GCS blobs.

### PR DESCRIPTION
**Pull request by udim**
See PR for sample outputs.

### CODE REVIEW

**Comment by udim** [comment]
R: @aaltay

File: sdks/python/apache_beam/internal/gcp/auth.py [hunk 1]
```diff
@@ -123,6 +123,7 @@ def get_service_credentials():
       logging.debug('Connecting using Google Application Default '
                     'Credentials.')
       return credentials
-    except Exception:
-      logging.warning('Unable to find default credentials to use.')
-      raise
+    except Exception as e:
+      logging.warning(
+          'Unable to find default credentials to use: %s\n'
+          'Connecting anonymously.' % e)
```
Comment (aaltay) [inline sdks/python/apache_beam/internal/gcp/auth.py]: Should we return an explicit `None` here?

Comment (udim) [inline sdks/python/apache_beam/internal/gcp/auth.py]: > **aaltay** wrote:
> Should we return an explicit `None` here?


Done.

**Comment by aaltay** [comment]
LGTM. I will merge after tests. Thank you.

**Pull request merged**